benchmark/buffers/buffer-bytelength.js

```diff
@@ -3,8 +3,8 @@ const common = require('../common');
 
 const bench = common.createBenchmark(main, {
   encoding: ['utf8', 'base64', 'buffer'],
-  len: [1, 2, 4, 16, 64, 256], // x16
-  n: [5e6]
+  len: [2, 16, 256], // x16
+  n: [4e6]
```

Comment:
There's very likely no need to have a lot of variation on the length here. So I removed the middle parts to check small buffers and bigger ones. The middle part should not be significant. The iterations are a bit lower as well but I wanted to still reach the CPU bound in most cases.